ccd-definition/AuthorisationCaseState.json

```diff
@@ -5,5 +5,12 @@
     "CaseStateID": "1_Initiation",
     "UserRole": "caseworker-publiclaw-localAuthority",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": 42736,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "CaseStateID": "2_Submitted",
```

Comment:
Do we wanna prefix states with numbers? That probably have no purpose. Let's remove.